<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IniciarSesion</class>
 <widget class="QDialog" name="IniciarSesion">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Iniciar Sesión</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>70</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblUsuario">
      <property name="text">
       <string>Usuario</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtUsuario"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lblClave">
      <property name="text">
       <string>Contraseña</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtClave">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>172</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushAceptar">
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushCancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushCancelar</sender>
   <signal>clicked()</signal>
   <receiver>IniciarSesion</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
